XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -38,6 +38,15 @@
                 <local:ControlExampleSubstitution Key="IsActive" Value="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" />
             </local:ControlExample.Substitutions>
 
+        </local:ControlExample>
```

Comment:
I think parts of the sample is missing, namely the sample and the options for said sample. This XAML wouldn't compile.

Reply:
@chingucoding that's weird I tried adding it again is it compiling this time? This is also using the 2.5 pre-release package.

Reply:
It's compiling fine now and works as expected, thank you!